docs/terms-of-service.rst

```diff
@@ -0,0 +1,656 @@
+but you may only use our content in the way we have allowed.*
+
+Read the Docs' rights to content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs and our licensors, vendors, agents, and/or our content providers
+retain ownership of all intellectual property rights of any kind related to our Websites and Services.
+We reserve all rights that are not expressly granted to you under this Agreement or by law.
+
+Read the Docs trademarks and logos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you'd like to use Read the Docs's trademarks,
+you must follow all of our `trademark guidelines`_.
+
+.. _trademark guidelines: https://read-the-docs-guidelines.readthedocs-hosted.com/
+
+
+API terms
+---------
+
+**Short version:** *You agree to these Terms of Service,
+plus this Section, when using any of Read the Docs' APIs (Application Provider Interface),
+including use of the API through a third party product that accesses Read the Docs.*
+
+No abuse or overuse of the API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Abuse or excessively frequent requests to Read the Docs via the API may result in the temporary or permanent suspension of your Account's access to the API.
+Read the Docs, in our sole discretion, will determine abuse or excessive usage of the API.
+We will make a reasonable attempt to warn you via email prior to suspension.
+
+You may not share API tokens to exceed Read the Docs' rate limitations.
+
+You may not use the API to download data or Content from Read the Docs for spamming purposes,
+including for the purposes of selling Read the Docs users' personal information, such as to recruiters, headhunters, and job boards.
+
+All use of the Read the Docs API is subject to these Terms of Service and our :doc:`privacy-policy`.
+
+Read the Docs may offer subscription-based access to our API for those Users who require high-throughput access
+or access that would result in resale of Read the Docs' Service.
+
+
+Additional terms for Documentation Sites
+----------------------------------------
+
+**Short version:** *Documentation Sites on Read the Docs are subject to certain rules,
+in addition to the rest of the Terms.*
+
+Documentation Sites
+~~~~~~~~~~~~~~~~~~~
+
+Each Read the Docs Account comes with the ability to host Documentation Sites.
+This hosting service is intended to host static web pages for All Users.
+Documentation Sites are subject to some specific bandwidth and usage limits,
+and may not be appropriate for some high-bandwidth uses or other prohibited uses.
+Read the Docs reserves the right at all times to reclaim any Read the Docs subdomain without liability.
+
+
+Third party applications
+---------------------------
+
+**Short version:** *You need to follow certain rules if you create an application for other Users.*
+
+Creating applications
+~~~~~~~~~~~~~~~~~~~~~
+
+If you create a third-party application or other developer product that collects User Personal Information
```

Comment:
"User Personal Information" was never defined and it might not be clear enough what this is.

Reply:
This term is described in detail in our privacy policy: https://docs.readthedocs.io/en/stable/privacy-policy.html#information-from-users-with-accounts